docs/faq.md

```diff
@@ -381,6 +381,18 @@ response headers. You can set cookies with the `Set-Cookie` response header—fo
 ctx.responseHeader.append("Set-Cookie", "foo=bar; Max=Age=120")
 ```
 
+### Access to fetch from origin has been blocked by CORS policy
+
+When a web page requests a resource from another origin, the restrictions for Cross-Origin Resource Sharing apply. 
+The server has to allow the client to send and receive protocol-specific headers. We show how to set up a server in the [CORS documentation](./cors.md).
+
+One specific gotcha to be aware of involves a new header `Connect-Protocol-Version`.  With the [v0.6.0 release](https://github.com/connectrpc/connect-es/releases/tag/v0.6.0) 
+of `@connectrpc/connect-web`, this header was added to all requests sent via the library. As a result, Connect clients
+that were working prior may start experiencing CORS errors upon upgrading to `v0.6.0`.  Note that this also only affects
+Connect traffic.  gRPC-Web traffic is unaffected.
```

Comment:
I'm wondering if this is even worth mentioning any more. We moved to `@connectrpc` after v0.13.0 and this predates even that by at least 7 releases. I'm sure people are still downloading this out in the wild, but is it worth the extra FAQ noise?

Reply:
True. There are only ~250 weekly downloads for < v0.6.0, and we link to the CORS docs from other FAQs. Updated.